src/neynarAPI/neynarV1API/swagger/spec.yaml

```diff
@@ -0,0 +1,1325 @@
+openapi: 3.0.0
```

Comment:
Should we add to `package.json` the script(s) used to generate the new typescript-axios code?

Reply:
Here's a super nitpicky nitpick, but I see they upgraded their specs from OpenAPI v2 to OpenAPI v3, which is much more associated with "OpenAPI" than "Swagger." Should we just rename and move everything over so we're only ever using "openapi" in the repo and avoid use of both terms at once?

Reply:
👍